functions/src/definitions/common/responseUtils.ts

```diff
@@ -840,6 +841,10 @@ async function respondToInstance(
         await sendTextMessage("user", from, responseText, data.id)
       }
   }
+
+  // Update repost channel
+  await repostUpdate(data.id, category)
```

Comment:
This should be removed based on earlier feedback? Cos not supposed to have more blasts triggered by replies